src/terrain/terrain_generation/land_generator.cpp

```diff
@@ -113,30 +127,49 @@ void LandGenerator::from_radius(int cr, int csr, TileStamp & ts) const {
     ts.y_end = volume[4];
     ts.z_end = volume[5];
 }
-void LandGenerator::from_grid(int cr, int csr, TileStamp &ts) const {
-    int x_center = (1 + 2 * (csr % data_[cr]["Grid"]["number"].asInt())) * ( data_[cr]["Grid"]["radius"].asInt() / data_[cr]["Grid"]["number"].asInt()) - data_[cr]["Grid"]["radius"].asInt();
-    int y_center = (1 + 2 * (csr / data_[cr]["Grid"]["number"].asInt())) * ( data_[cr]["Grid"]["radius"].asInt() / data_[cr]["Grid"]["number"].asInt()) - data_[cr]["Grid"]["radius"].asInt();
+
+void
+LandGenerator::from_grid(int cr, int csr, TileStamp& ts) const
+{
+    int x_center = (1 + 2 * (csr % data_[cr]["Grid"]["number"].asInt()))
+                       * (data_[cr]["Grid"]["radius"].asInt()
+                          / data_[cr]["Grid"]["number"].asInt())
+                   - data_[cr]["Grid"]["radius"].asInt();
+    int y_center = (1 + 2 * (csr / data_[cr]["Grid"]["number"].asInt()))
+                       * (data_[cr]["Grid"]["radius"].asInt()
+                          / data_[cr]["Grid"]["number"].asInt())
+                   - data_[cr]["Grid"]["radius"].asInt();
```

Comment:
wow linting has really gotten good. this is super useful I like how this is written.